components/collapsible/accordion/src/index.js

```diff
@@ -16,6 +16,9 @@ class CollapsibleAccordion extends Component {
 
   _collapseItems (collapsed, id) {
     this.setState({ openIndex: id === this.state.openIndex ? null : id })
+    if (this.props.onItemChange) {
```

Comment:
Better if you put this.props.onItemChange value as an empty function in the defaultProps and avoid using a conditional.

Reply:
@miduga good! Done!